azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +194,43 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "dst",
+							ValidateFunc: validation.StringInSlice([]string{
+								"dst",
+							}, true),
+						},
+
+						"remote_volume_location": azure.SchemaLocation(),
+
+						"remote_volume_resource_id": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: azure.ValidateResourceID,
+						},
+
+						"replication_schedule": {
```

Comment:
given this is not so much a schedule as a frequency 
```suggestion
						"replication_frequency": {
```

Reply:
This one will need to stay as is since our documentation, portal, sdk and other tools references this object by that name.